tutorial/index.md

```diff
@@ -79,48 +82,38 @@ juju offer self-signed-certificates:send-ca-cert
 
 Running `juju status` should now two offers that we will use from a different model in the next step.
 
-## Setup External IdP
-
-We recommend using the following [how-to](https://charmhub.io/topics/canonical-identity-platform/how-to/integrate-external-identity-provider)
-and choosing your preferred identity provider.
-
-```{note}
-You can temporarily skip this step and return to it later, JIMM can still run without integrating the identity bundle to an external identity provider but login to JIMM will not work.
+### Create an user.
+```text
+# disable MFA to avoid unnecessary steps
+juju config kratos enforce_mfa=False
+# create the user and get the identity-id
+juju run kratos/0 create-admin-account email=test@example.com password=test username=admin
+# reset the password to make it active
+juju add-secret password-secret password=abc
+juju grant-secret password-secret kratos
+juju run kratos/0 reset-password identity-id=<identity-id> password-secret-id=<secret:id>
 ```
 
-Setting up an IdP to point to your local environment can be tricky depending on the provider used. Below are some tips to make this work.
-
-When setting up certain providers, e.g. Google, security restrictions limit what redirect URLs can be used. Some restrictions include,
-
-- The redirect URL must be `https`.
-- The redirect URL must be a top level domain `.com`.
-- The redirect URL cannot be an IP address.
-
-The redirect URL is the URL that your browser is returned to after you have signed in at the identity provider. When using Canonical's
-identity bundle, the redirect URL after login will be something like `https://<kratos-public-url>/self-service/methods/oidc/callback/<provider-id>`.
-as described in the above how-to. Although the URL is `https`, it is an IP address.
-This address needs to be registered in your identity provider as an approved redirect URI/URL.
-
-If your preferred identity provider does not accept an IP address, we recommend using a tool like `https://nip.io/`,
-a DNS resolver service that can map any IP address to a hostname.
-
-| This service can map hostnames of the form `<anything>[.-]<IP Address>.nip.io` to return simply `<IP Address>`.
-| E.g. `magic.127.0.0.1.nip.io` resolves to `127.0.0.1`. This service is very useful when working with an IdP locally for testing.
+### Expose the identity bundle to your host machine.
```

Comment:
This should have a note  below that it's only necessary if running in Multipass.